pkg/apis/operator/v1alpha1/tektonchain_validation.go

```diff
@@ -68,6 +68,28 @@ func (tcs *TektonChainSpec) ValidateChainConfig(path string) (errs *apis.FieldEr
 		}
 	}
 
+	if tcs.ArtifactsPipelineRunFormat != "" {
+		if tcs.ArtifactsPipelineRunFormat != "in-toto" {
```

Comment:
the `slsa/v1` is also allowed i guess

Reply:
Its allowed for backwards compatibility. should it also be included?

Reply:
We have a follow up for it which addresses the newly added values and some more new fields

Reply:
@sabre1041 it is like that because that was the only supported value when these config were added